github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -315,6 +315,100 @@ presubmits:
           path: /dev
           type: Directory
         name: devices
+  - always_run: false
+    skip_report: true
```

Comment:
I think it is worth setting `skip_report: false` for both presubmits as it is nice to get the feedback on the PR rather than having to find the job in the prow dashboard.